array-methods-more/get-full-names.js

```diff
@@ -2,13 +2,23 @@
   Refactor this function to use .map and ES6 syntax
  */
 
+// function getFullNames(persons) {
+//   const result  = [];
+//   for (var i = 0; i < persons.length; i++) {
+//     var person = persons[i];
+//     var fullName = person.firstName + ' ' + person.lastName;
+//     result.push({ fullName: fullName });
+//   }
+//   return result;
+// };
+
+// module.exports = getFullNames;
+
 function getFullNames(persons) {
-  const result  = [];
-  for (var i = 0; i < persons.length; i++) {
-    var person = persons[i];
-    var fullName = person.firstName + ' ' + person.lastName;
-    result.push({ fullName: fullName });
-  }
+  var result = persons.map(function(person){
+var fullName = person.firstName + ' ' + person.lastName;
+    return {fullName: fullName };
```

Comment:
Little trick: with ES6 you can just write `{ fullName }` if the key has the same name as value (where value is a reference to a variable). Google object literal property value shorthand for more info.